### PR TITLE
Release/v0.0.1

### PR DESCRIPTION
**Pull request by juliomoreira**

### CODE REVIEW

File: react/components/withProductContext.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React, { useContext } from 'react'
+import { ProductContext } from 'vtex.product-context'
+
+function withProductContext(Component) {
```
Comment (jgfidelis) [inline react/components/withProductContext.js:4]: this is a hook, it should not be called "with Product context", its breaking the pattern. A better name would be "useProductContext"

**Comment by jgfidelis** [comment]
I also don't  think you should be releasing a major, the first version can be a 0.x.

File: react/components/withProductContext.js (hunk 1 above)
Comment (klzns) [reply react/components/withProductContext.js:4]: That's a HOC abstracting a hook to a class component, it's ok

File: store/interfaces.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "flixmedia": {
```
Comment (klzns) [inline store/interfaces.json]: What do you think of extending the interface `product-description`?

Example: `product-description.flixmedia`

That way we don't need to allow a new block in the store and I think it makes sense semantically.

**Review by klzns — CHANGES_REQUESTED** [review]

File: store/interfaces.json (hunk 2 above)
Comment (juliomoreira) [reply store/interfaces.json]: @klzns  the `product-description` is being used, how can we proceed in this case?

Comment (klzns) [reply store/interfaces.json]: You can use both. No problem.

blocks.json
```js
"store.product": {
    "children": [
      "product-description",
      "product-description.flixmedia",
    ]
}
```

Comment (juliomoreira) [reply store/interfaces.json]: Ok @klzns!

**Review by klzns — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,19 +1,44 @@
...
+To configure or customize this app, you need to import it in your dependencies in `manifest.json`.
+
+```json
+  dependencies: {
+    "vtex.flixmedia": "1.x"
+  }
 ```
 
-Open your admin in the Apps section:
+Then, add `product-details` block into your `interfaces.json` theme as we do in our [Store theme app](https://github.com/vtex-apps/store-theme/blob/master/store/blocks.json).
```
Comment (klzns) [inline README.md]: We should update the README.

Comment (juliomoreira) [reply README.md]: Done @klzns

**Review by klzns — DISMISSED** [review]

**Review by klzns — APPROVED** [review]

**Review by jgfidelis — APPROVED** [review]

**Pull request merged**